<launch>
    <include file="$(find lidar_odometry_mapping)/launch/loam_velodyne_HDL_64_kitti.launch" />

    <param name="detach_VO_LO" type="bool" value="true" />
    <param name="reset_VO_to_identity" type="bool" value="false" />
    <param name="remove_VO_outlier" type="int" value="100" />
    <param name="keypoint_NMS" type="bool" value="false"/>
    <param name="CLAHE" type="bool" value="false"/>
    <param name="visualize_optical_flow" type="bool" value="true" />
    <param name="optical_flow_match" type="bool" value="false" />

    <node pkg="vloam_main" type="vloam_main_node" name="vloam_main_node" output="screen">
        <param name="rosbag_rate" type="double" value="0.33" />
        <param name="visualize_depth" type="bool" value="false" />
        <param name="publish_point_cloud" type="bool" value="true" />
        <param name="save_traj" type="bool" value="true" />
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find vloam_main)/rviz/vloam.rviz" />

</launch>